test/gurka/gurka.h

```diff
@@ -166,12 +166,23 @@ build_config(const std::string& tiledir,
   return ptree;
 }
 
+midgard::PointLL to_ll(const nodelayout& nodes, const std::string& node_name) {
+  return nodes.at(node_name);
+}
+std::vector<midgard::PointLL> to_lls(const nodelayout& nodes,
+                                     const std::vector<std::string>& node_names) {
+  std::vector<midgard::PointLL> lls;
+  for (const auto& node_name : node_names)
+    lls.emplace_back(to_ll(nodes, node_name));
+  return lls;
+}
```

Comment:
small refactor to make it possible to build a valhalla request with lls instead of OSM node names. didnt actually expose this in the match or route method but we'll see what the future holds